instrumentation/opentelemetry-instrumentation-asgi/src/opentelemetry/instrumentation/asgi/__init__.py

```diff
@@ -446,11 +446,13 @@ def set_status_code(
         status_code = -1
     if metric_attributes is None:
         metric_attributes = {}
+    status_server_span = True
     _set_status(
         span,
         metric_attributes,
         status_code,
         status_code_str,
+        status_server_span,
```

Comment:
Nit: You can just pass in the variable inline, I believe it is more python to do so. 

```
_set_status(
...
server_span=True,
sem_conv_opt_in_mode=sem_conv_opt_in_mode,
)